app/models/north_utility.rb

```diff
@@ -1,2 +1,9 @@
 class NorthUtility < Utility
+  def short_content
```

Comment:
```suggestion
  def short_content_length
```
Que te parece este nombre? aplica para ambas Utilities

Reply:
Dale, me parece perfecto!